apps/ledger-live-desktop/src/renderer/components/Onboarding/Screens/Tutorial/index.tsx

```diff
@@ -590,19 +638,58 @@ export default function Tutorial({ useCase }: Props) {
             ? history.push(recoverDiscoverPath)
             : history.push("/onboarding/select-use-case"),
       },
-    ],
-    [
-      userChosePinCodeHimself,
-      useCase,
-      userUnderstandConsequences,
-      connectedDevice,
-      history,
-      path,
-      fromRecover,
-      recoverDiscoverPath,
-      dispatch,
-    ],
-  );
+      {
+        id: ScreenId.secureYourCrypto,
+        component: SecureYourCrypto,
+        useCases: [OnboardingUseCase.setupDevice],
+        next: () => {
+          dispatch(
+            setIsOnboardingReceiveFlow({
+              isFlow: true,
+              isSuccess: false,
+            }),
+          );
+          openAssetFlow();
+        },
+        nextSecondary: () => history.push(`${path}/${ScreenId.welcomeToWalletWithoutFunds}`),
+        previous: () => history.push(`${path}/${ScreenId.genuineCheck}`),
+      },
+      {
+        id: ScreenId.welcomeToWalletWithFunds,
+        component: WelcomeToWalletWithFunds,
+        useCases: [OnboardingUseCase.setupDevice],
+        next: completeOnboarding,
+      },
+      {
+        id: ScreenId.welcomeToWalletWithoutFunds,
+        component: WelcomeToWalletWithoutFunds,
+        useCases: [OnboardingUseCase.setupDevice],
+        next: completeOnboarding,
+      },
+    ];
+    return unfilteredScreens.filter(
+      ({ id }) =>
+        !!nanoOnboardingFundWalletFeature ||
+        ![
+          ScreenId.secureYourCrypto,
+          ScreenId.welcomeToWalletWithFunds,
+          ScreenId.welcomeToWalletWithoutFunds,
+        ].includes(id),
+    );
```

Comment:
Easier to read and more performant like this:
```suggestion
    return !!nanoOnboardingFundWalletFeature ? unfilteredScreens : unfilteredScreens.filter(
      ({ id }) =>
        ![
          ScreenId.secureYourCrypto,
          ScreenId.welcomeToWalletWithFunds,
          ScreenId.welcomeToWalletWithoutFunds,
        ].includes(id),
    );
```